src/Logging/HTTPOutputHandler.php

```diff
@@ -70,6 +72,38 @@ public function setStatusCode($statusCode)
     }
 
     /**
+     * Set a formatter to use if Director::is_cli() is true
+     */
+    public function setCliFormatter(FormatterInterface $cliFormatter)
```

Comment:
Our current convention is to fully capitalise initials. Should this be `setCLIFormatter`?